deploy-board/deploy_board/webapp/helpers/environs_helper.py

```diff
@@ -65,6 +65,45 @@
 # Nimbus-related helpers
 
 
+def create_identifier_for_new_stage(request, env_name, stage_name):
+    """ Create a Nimbus Identifier for the new stage. Assumes that the environment has at least one stage with externalId set.
+        This is needed so that the method knows which project to associate the new stage to.
+
+        If the environment has no stage with externalId set, this method will not attempt to create an Identifier.
+    """
+    # Only create identifier for Pinterest environments
+    if not IS_PINTEREST:
```

Comment:
could do same for delete_nimbus_identifier. 

This methods could be part of interface to avoid `if IS_PINTEREST` everywhere but not in the scope of this PR.

Reply:
Moved `IS_PINTEREST` into `delete_nimbus_identifier`.

yeah would be nice to avoid `IS_PINTEREST`